src/darwin/Framework/CHIP/MTROTAImageTransferHandler.mm

```diff
@@ -310,9 +310,7 @@ - (instancetype)initWithMTROTAImageTransferHandler:(MTROTAImageTransferHandler *
 
     TransferSession::OutputEventType eventType = event.EventType;
 
-    if (eventType != TransferSession::OutputEventType::kNone) {
-        ChipLogError(BDX, "OutputEvent type: %s", event.ToString(eventType));
-    }
+    ChipLogDetail(BDX, "OutputEvent type: %s", event.ToString(eventType));
```

Comment:
Followup: this is in fact somewhat spammy, in the sense of "one line for each block we receive".  But for now, fine, especially with the level downgrade.

Reply:
Yeah i kept this to make sure the pipe works. This is relatively new code. I did think of removing it but for now I can trace the darwin code got it.

Reply:
Filed https://github.com/project-chip/connectedhomeip/issues/37419 to track